src/modules/UI/components/Slider/Slider.js

```diff
@@ -70,7 +70,10 @@ export const SliderComponent = (props: Props) => {
     setCompleted(true)
   }
 
-  if (reset && translateX.value === 0) resetSlider()
```

Comment:
Are you sure that getting rid of the condition `translateX.value === 0` won't cause a problem?
If I'm not mistaken, it was to prevent reseting a slider before it reaches the end.